code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -411,6 +417,7 @@
 
 /obj/item/ammo_magazine/ammobox/kurtz_50/rubber
 	name = "ammunition box (12mm rubber)"
+	desc = "A box of 7.62mm rifle ammunition. 180ct, deforming traumatic rounds. You can see caliber information on the label. It contains a lengthy warning label clearly stating it is a less-lethal round and declaring the manufacturer is not responsible for damages done"
```

Comment:
```suggestion
	desc = "A box of 7.62mm rifle ammunition. 180ct, deforming impact rounds. You can see caliber information on the label. It contains a lengthy warning label clearly stating it is a less-lethal round and declaring the manufacturer is not responsible for damages done"
```

Reply:
suggest changing this to 'Deforming Impact Round' just to be clear. The desc makes it sound like it's frangible/HP rather than rubber.